content/en/news/releases/1.13.x/announcing-1.13/change-notes/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Announcing Istio 1.13.0
+linktitle: 1.13.0
+subtitle: Minor Release
+description: Istio 1.13.0 minor release.
+publishdate: 2021-02-08
+release: 1.13.0
+aliases:
+    - /news/announcing-1.13.0
+---
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+## Traffic Management
+
+- **Added** an API (CRD) for configuring `ProxyConfig` values containing a stable subset of the configuration from `MeshConfig.DefaultConfig`.
+
```

Comment:
Here between "Added" and "Promoted", there are two empty lines. On line 43 to 45, between two "Added", there is only one empty line. Should the number of empty lines between items be consistent?

Reply:
They should be 1 empty line. I put up suggestions.

Reply:
```suggestion
```